libs/SmartStore/src/com/salesforce/androidsdk/smartstore/store/DBOpenHelper.java

```diff
@@ -237,6 +258,11 @@ public static synchronized void deleteDatabase(Context ctx, String dbNamePrefix,
 				communityDBNamePrefix.append(accountSuffix);
 		    	deleteFiles(ctx, communityDBNamePrefix.toString());
 			}
+
+			// Delete external blobs directory
+			StringBuilder blobsDbPath = new StringBuilder(ctx.getApplicationInfo().dataDir);
+			blobsDbPath.append("/databases/").append(fullDBName).append(EXTERNAL_BLOBS_SUFFIX);
+			removeAllFiles(new File(blobsDbPath.toString()));
```

Comment:
@bhariharan external storage is removed here in deleteDatabase along when the .db file itself is removed